README.md

```
@@ -65,7 +65,7 @@ export KUBE_CONTEXT=<context-within-the-kubeconfig-to-use>
 | :-------------: | :--------------------------------------------------------: | :--------------: |
 |     _helm_chart_path_     |             path to Helm chart to use for the service deployment/release              |    `../../helm`    |
 |     _helm_namespace_      |  Kubernetes namespace to deploy to   |      `default`       |
-|    _helm_values_path_     | file to load Helm chart values (see [here](./helm/README.md) for available values) |       `values.yml`       |
+|    _helm_values_path_     | file to load Helm chart values (see [here](./helm/README.md) for available values) |       `values.yaml`       |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)